spec/Candid.md

```diff
@@ -1177,7 +1177,7 @@ M(_ : reserved) = .
 M : <val> -> <constype> -> i8*
 M(null : opt <datatype>) = i8(0)
 M(?v   : opt <datatype>) = i8(1) M(v : <datatype>)
-M(v*   : vec <datatype>) = leb128(N) M(v : <datatype>)*
+M(v*   : vec <datatype>) = leb128(|v*|) M(v : <datatype>)*
```

Comment:
```suggestion
M(v^N  : vec <datatype>) = leb128(N) M(v : <datatype>)^N
```
Seems to be the idiom used elsewhere (e.g. in T*)

Reply:
But there, N is bound on the lhs. Here it ain't. Just a nit really.

Reply:
That works for me.